lib/routes/account.js

```diff
@@ -18,16 +21,28 @@ module.exports = function (
   db,
   mailer,
   Password,
-  redirectDomain,
-  verifierVersion,
-  isProduction,
-  domain,
-  resendBlackoutPeriod,
+  config,
   customs,
   isPreVerified,
   checkPassword
   ) {
 
+  var openIdExtensions = [
```

Comment:
Looking at other files in this repo, we have a very slight convention of capitalizing module-level constants, e.g. `OPENID_EXTNSIONS`.  Although I'm not sure if that's just me doing out of habit and nobody else complaining...